pom.xml

```diff
@@ -112,29 +112,6 @@
     <build>
         <pluginManagement>
             <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-enforcer-plugin</artifactId>
-                    <executions>
-                        <execution>
-                            <id>enforce-java-version</id>
-                            <goals>
-                                <goal>enforce</goal>
-                            </goals>
-                        </execution>
-                    </executions>
-                    <configuration>
-                        <rules>
-                            <!-- JDK 1.8 is required to build Mojarra 2.3.x. -->
-                            <requireJavaVersion>
-                                <message>
-                                    To build this project, JDK 1.8 is required. Please install it.
-                                </message>
-                                <version>[1.8,1.9)</version>
-                            </requireJavaVersion>
-                        </rules>
-                    </configuration>
-                </plugin>
```

Comment:
If you remove this you need something to enforce the use of SE 8 when releasing. For example:

https://github.com/wildfly/wildfly/blob/main/pom.xml#L1133-L1146

So I think you need to leave the plugin in place but just tweak the execution by moving the configuration into it and setting the phase to 'deploy'.